README.md

```diff
@@ -1,5 +1,11 @@
 # build-your-own-internet
 
+## How to approach this repo
+The intention behind this project is discovery and understanding. The idea is to explore how the internet is put together and use the common tools the industry uses to see and understand how servers are communicating with each other. 
```

Comment:
`servers are communicating with each other` ... 🤔 not sure about this since it's going to be largely about routers communicating with each other, clients communicating with DNS and servers and caches, etc. Maybe 

`The idea is to explore how the internet is put together and use common industry tools to see and understand all the communications necessary to make the services and apps we use work.` or something like that?